### DISCUSSION TITLE
The crack surface density function

### DISCUSSION THREAD

**Discussion by ghost** [idaholab/moose#26514] [Q&A Modules: Phase field]
Hello, 

Is there a way to extract the crack surface density (γ(d,∇d)  https://mooseframework.inl.gov/source/materials/ComputeLinearElasticPFFractureStress.html)?

**Comment by jiangwen84** [comment]
You can write a postprocessor class to output the quantity.

**Reply by ghost** [reply]
Hi, I want to extract A(d) to be more precise, the crack surface integral over the whole volume, to measure the crack surface generated during the loading at the end of a test. I know that I can use postprocessor classes (such as ElementIntegralVariablePostprocessor) to integrate the variable gamma (the crack surface density) but where can I find the crack surface density that is function of the square of the gradient of d and also the local term function of d? Thanks!
